src/steadystateproblem.cpp

```diff
@@ -90,16 +90,21 @@ void SteadystateProblem::workSteadyStateBackwardProblem(
 void SteadystateProblem::findSteadyState(const Solver *solver, Model *model,
                                          int it) {
     steady_state_status_.resize(3, SteadyStateStatus::not_run);
+    bool turnOffNewton = solver->getSensitivityMethod() ==
```

Comment:
Because in adjoint case steady-state computation part is independent of sensitivities computation.
If `model->getSteadyStateSensitivityMode() == SteadyStateSensitivityMode::integrationOnly` in adjoint case, I think model steady-state could still be found by Newton method.

Reply:
but that also excludes `solver->getSensitivityMethod()==SensitivityMethod::none` or situations where `solver->getSensitivityOrder() < SensitivityOrder::first` where anyways no sensitivities are computed. It also ignores `solver->getSensitivityMethodPreequilibration()`.

 I think the best way is to use SteadystateProblem::getSensitivityFlag here.